pages/pokedex/index.tsx

```diff
@@ -1,15 +1,17 @@
-import { gql } from '@apollo/client';
-import { getApolloClient } from '../../apollo-client';
-import type { InferGetServerSidePropsType, NextPage } from 'next';
-import type { Pokemon } from '..';
-import { CardGrid } from '../../components';
-import styles from '../../styles/Page.module.css';
-import { PokemonCard } from '../../components';
-import { POKEMON } from '../../config';
+import { gql } from "@apollo/client";
+import { getApolloClient } from "../../apollo-client";
+import type { InferGetServerSidePropsType, NextPage } from "next";
+import { CardGrid } from "../../components";
+import styles from "../../styles/Page.module.css";
+import { PokemonCard } from "../../components";
+import { POKEMON } from "../../config";
 
 type Pokemons = {
```

Comment:
Type the return value in the file, rather than importing the other type value - this isn't [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), but will make more sense later.